pkg/kademlia/kademlia_planet_test.go

```diff
@@ -38,13 +39,17 @@ func TestRequestInfo(t *testing.T) {
 		SatelliteCount: 1, StorageNodeCount: 1, UplinkCount: 0,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
 		node := planet.StorageNodes[0]
-		info, err := planet.Satellites[0].Kademlia.Service.FetchInfo(ctx, node.Local())
+		info, err := planet.Satellites[0].Kademlia.Service.FetchInfo(ctx, node.Local().Node)
 		require.NoError(t, err)
 		require.Equal(t, node.Local().Type, info.GetType())
-		require.Equal(t, node.Local().Metadata.GetEmail(), info.GetOperator().GetEmail())
-		require.Equal(t, node.Local().Metadata.GetWallet(), info.GetOperator().GetWallet())
-		require.Equal(t, node.Local().Restrictions.GetFreeDisk(), info.GetCapacity().GetFreeDisk())
-		require.Equal(t, node.Local().Restrictions.GetFreeBandwidth(), info.GetCapacity().GetFreeBandwidth())
+		require.Equal(t, node.Local().Operator.GetEmail(), info.GetOperator().GetEmail())
```

Comment:
Doesn't `require.Equal(t, node.Local(), info)` do the same thing?

Reply:
No. Comparing protobufs usually fails due to different values in hidden XXX fields.

Reply:
`require.Empty(t, cmp.Diff(a, b, cmp.Comparer(pb.Equal)))`

Reply:
I will check this spell.

Reply:
I could not use this directly to compare `node.Local()` to `info` because they are of different types (`NodeDossier` vs `InfoResponse`). But they worked for simplifying the checks for the `Operator`, `Capacity` and `Version` fields. Thanks for the hint!